oss-cloud-frontend/src/main.js

```diff
@@ -5,9 +5,11 @@ import VueRouter from "vue-router";
 
 import routes from "./routes";
 
+
```

Comment:
Please dont insert unnecessary spacing. Usually one empty line is used.
There are multiple empty lines occurrences in `Header.vue` template section as well.
We can add https://eslint.org/docs/rules/no-multiple-empty-lines with max:1, but I'm not sure where it's configured when using vue-cli-service for linting.